support/fixdeps.js

```diff
@@ -30,6 +30,43 @@ if (!fs.existsSync(expected)) {
 	var actualPath = path.dirname(require.resolve(path.join(dependency, 'package.json')));
 
 	if (actualPath.indexOf(expectedPath) !== 0) {
-		fs.symlinkSync(path.relative(path.dirname(expectedPath), actualPath), expectedPath, 'junction');
+		try {
+			fs.symlinkSync(path.relative(path.dirname(expectedPath), actualPath), expectedPath, 'junction');
+		}
+		catch (error) {
+			console.warn('Symlinking %s to %s failed with %s. Copying instead...', actualPath, expectedPath, error.code);
+			copy(actualPath, expectedPath);
+		}
 	}
 });
+
+function copy(source, target) {
+	try {
+		var stats = fs.statSync(source);
+	}
+	catch (error) {
+		if (error.code !== 'ENOENT') {
+			throw error;
+		}
+
+		return;
+	}
+
+	if (stats.isDirectory()) {
+		try {
+			fs.mkdirSync(target);
+		}
+		catch (error) {
+			if (error.code !== 'EEXIST') {
```

Comment:
This isn't strictly necessary, since even reinstalling will uninstall first (and thus delete the node_modules directory), so this directory cannot exist.